build.gradle

```diff
@@ -180,6 +180,10 @@ project('segmentstore:storage:impl') {
         testCompile group: 'org.apache.hadoop', name: 'hadoop-minicluster', version: hadoopVersion, withoutLog4j
         testCompile group: 'ch.qos.logback', name: 'logback-classic', version: qosLogbackVersion
 
+        //For Extended S3
+        compile "com.emc.ecs:object-client:3.0.0"
```

Comment:
This is an open source library available at https://github.com/EMCECS/ecs-object-client-java.git.